src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -14,13 +14,55 @@ public sealed partial class JsonDocument
     {
         private const int UnseekableStreamInitialRentSize = 4096;
 
+        /// <summary>
+        ///   Parse memory as UTF-8-encoded text representing a single JSON value into a JsonDocument.
+        /// </summary>
+        /// <remarks>
+        ///   The <see cref="ReadOnlyMemory{T}"/> value will be used for the entire lifetime of the
+        ///   JsonDocument object, and the caller must ensure that the data therein does not change during
+        ///   the object lifetime.
+        ///
+        ///   Because the input is considered to be text, a UTF-8 Byte-Order-Mark (BOM) must not be present.
```

Comment:
📝 This is missing `<para>` elements. Combine the sentences into one paragraph, or wrap both to indicate they are separate.